src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  let partnerSex = 'husband`s';
+
+  if (sex === 'm') {
+    partnerSex = 'wife`s';
+  }
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">
+          My name is
+          {' '}
+          {name}
+        </h2>
+        <p className="Person__age">{age && `I am ${age}`}</p>
```

Comment:
No, wrap your whole element with `age &&` condition, you don't need element with blank content